README.md

```diff
@@ -1,8 +1,89 @@
+  - JDBC App: Designed, implemented, tested, and deployed a Java app that uses the JDBC library, the DAO design pattern, a Docker PostgreSQL database, IntelliJ IDE, and DBeaver client to CRUD persisted sales data, resulting in atomic business-related queries.
+  - Grep App: Designed, implemented, tested, and deployed a Java implementation of the Linux grep command using features/libraries (like regex, lambda, Stream APIs, and JUnit), which recursively traverses file paths resulting in an output file with all matching strings.
+
+**Springboot App** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_VishaySingh/tree/master/springboot)]: Designed, implemented, tested, and deployed a backend trading app that can create
+and manage clients/accounts, and execute security orders on stocks defined by IEX Cloud's REST 
+API. This app leverages three-tier architecture, microservices, and DAO design patterns. It was written
+with standard Java libraries, SpringBoot Inversion of Control, SwaggerUI, PostgreSQL external 
+databases, Maven, and deployed using Docker.
+
+
+**Python Data Analytics** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_VishaySingh/tree/master/python_data_anlytics)]: Not started.
+
+**Hadoop** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_VishaySingh/tree/master/hadoop)]: Not started.
+
+**Spark** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_VishaySingh/tree/master/spark)]: Not started.
+
+**Cloud/DevOps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_VishaySingh/tree/master/cloud_devops)]: Not started.
+
+
+## Highlighted Projects
+**Google Developer Student Club Website** [[GitHub](https://github.com/UTSCCSCC01/finalprojectw22-GDSC2.0)]: Designed, implemented, and tested an MVC architecture web application using the MERN 
+stack (MongoDB, Express.js, React.js, Node.js) in a team of 6.
+
+
+**Deep Learning - RNN Lyric Generation Model** [[GitHub](https://github.com/V35games/Lyric-Generation-Model)]: Designed, implemented, trained, and tested an RNN Generative Model to achieve a 58% test accuracy.
+
+
+**Information Security - Rootkit Lecture and Tutorial** [[GitHub](https://github.com/V35games/CSC427-LD_PRELOAD-Rootkit)]: Collaborated with a partner to create an in-depth presentation, lab, and assignment about rootkits 
+that exploit the Linux LD_PRELOAD environment variable.
+
+
+
+## Professional Experiences
+
+**Software Engineer (Bilingual / Entry Level), Jarvis (2022-present)**: Designed, implemented, tested, and deployed software projects relating to Linux/Bash scripting, 
+SQL, and Java, using Agile/Scrum methodology, Git Flow, and Docker. Attended and passed Master 
+Code Review meetings that certify acceptable code quality and requisite knowledge. Lead daily 
+stand-up meetings during the first and fifth sprints, which coordinated and tracked scrum team progress.
```

Comment:
Don't specify which sprints you lead. This is one of those scenarios where less is more.